hack/packages/scripts/package-utils.sh

```diff
@@ -8,86 +8,106 @@ set -eoux pipefail
 PROJECT_ROOT=$(git rev-parse --show-toplevel)
 TOOLS_DIR="${PROJECT_ROOT}/hack/tools"
 TOOLS_BIN_DIR="${TOOLS_DIR}/bin"
-MANAGEMENT_PACKAGES_BUILD_ARTIFACTS_DIR="${PROJECT_ROOT}/build"
-MANAGEMENT_PACKAGE_REPOSITORY_DIR="${PROJECT_ROOT}/management-packages"
+PACKAGES_BUILD_ARTIFACTS_DIR="${PROJECT_ROOT}/build"
 
-if [[ ${MANAGEMENT_PACKAGE_REPO_VERSION:0:1} == "v" ]] ; then
-  REPO_VERSION=${MANAGEMENT_PACKAGE_REPO_VERSION:1}
-else
-  REPO_VERSION=${MANAGEMENT_PACKAGE_REPO_VERSION}
+if [[ ${REPO_VERSION:0:1} == "v" ]] ; then
+  REPO_VERSION=${REPO_VERSION:1}
 fi
 
 function generate_single_imgpkg_lock_output() {
-	path=${MANAGEMENT_PACKAGE_REPOSITORY_DIR}/${name}
+	path="${PROJECT_ROOT}/packages/${PACKAGE_REPOSITORY}/${PACKAGE_NAME}"
 	mkdir -p "$path/bundle/.imgpkg"
 	yttCmd="${TOOLS_BIN_DIR}/ytt --ignore-unknown-comments -f $path/bundle/config/"
-	${yttCmd} | "${TOOLS_BIN_DIR}"/kbld -f - -f "${PROJECT_ROOT}/management-packages/kbld-config.yml" --imgpkg-lock-output "$path/bundle/.imgpkg/images.yml" > /dev/null
+	${yttCmd} | "${TOOLS_BIN_DIR}"/kbld -f - -f "${PROJECT_ROOT}/packages/kbld-config.yml" --imgpkg-lock-output "$path/bundle/.imgpkg/images.yml" > /dev/null
```

Comment:
This needs to change as well: `kbld-config.yml` to `kbld-config.yaml`. And in other places as well.

Reply:
thanks for catching that, i forgot to squash and push that commit..